mp4parse/tests/public.rs

```diff
@@ -32,6 +32,7 @@ static IMAGE_AVIF_CORRUPT: &str = "tests/bug-1655846.avif";
 static IMAGE_AVIF_CORRUPT_2: &str = "tests/bug-1661347.avif";
 static IMAGE_AVIF_GRID: &str = "av1-avif/testFiles/Microsoft/Summer_in_Tomsk_720p_5x4_grid.avif";
 static AVIF_TEST_DIR: &str = "av1-avif/testFiles";
+static VIDEO_H263_3GP: &str = "tests/bbb_sunflower_QCIF_30fps_h263_noaudio_1f.3gp";
```

Comment:
Could you add a brief comment describing how the file was created to help with reproducing similar files if needed?

Reply:
Sure, just add the comment in the updated commit, thanks.

Reply:
Indeed, the command is more generic and way better, thanks!
I've updated this information and new test file in the latest commit.
Thanks again!